cpp/tests/large_strings/parquet_tests.cpp

```diff
@@ -69,3 +70,57 @@ TEST_F(ParquetStringsTest, ReadLargeStrings)
   // go back to normal threshold
   unsetenv("LIBCUDF_LARGE_STRINGS_THRESHOLD");
 }
+
+TEST_F(ParquetStringsTest, ChunkedReadLargeStrings)
+{
+  // Construct a table with one large strings column > 2GB
+  auto const wide = this->wide_column();
+  auto input      = cudf::concatenate(std::vector<cudf::column_view>(120000, wide));  ///< 230MB
+
+  int constexpr multiplier = 12;
+  std::vector<cudf::column_view> input_cols(multiplier, input->view());
+  auto col0 = cudf::concatenate(input_cols);  ///< 2.70GB
+
+  // Expected table
+  auto const expected    = cudf::table_view{{col0->view()}};
+  auto expected_metadata = cudf::io::table_input_metadata{expected};
+  expected_metadata.column_metadata[0].set_encoding(
+    cudf::io::column_encoding::DELTA_LENGTH_BYTE_ARRAY);
```

Comment:
Why is the encoding specified for this test?
Apologies if I missed a conversation :)

Reply:
Came from copy-pasting the test body from the above GTest which does the same thing as this one except using `read_parquet`.

Reply:
Update: Removing this encoding changes the chunking behavior and requires empirically choosing another smaller `pass_read_limit` so leaving this as is.

Reply:
Same comment as above. Disabled by default with comments added.